python/cudf/cudf/core/index.py

```diff
@@ -279,13 +279,15 @@ def rename(self, name):
         -------
         Index
 
-        Difference from pandas:
-          * Not supporting: inplace
         """
-        out = self.copy(deep=False)
-        out.name = name
+        if inplace is True:
+            self.name = name
+            return self
```

Comment:
If `inplace=True` should it return self or None?

Reply:
pandas seems to return the actual object here

>>> ix = pd.Index([1,2,3], name='one')
>>> ix_rename = ix.rename('two')
>>> ix_rename
Int64Index([1, 2, 3], dtype='int64', name='two')

Reply:
This isn't with the inplace argument which I believe makes it return None.

Reply:
Ah yes. Updated